.gitignore

```diff
@@ -165,5 +165,5 @@ cython_debug/
 
 # rialto-airflow
 data/
+config/*-key.json
```

Comment:
this is where the google service account json file is going to live, we don't want to check it in